pyproject.toml

```diff
@@ -201,6 +200,5 @@ required-imports = ["from __future__ import annotations"]
     "N801", # Class casing
     "PLR2004", # Magic numbers
 ]
-"onnx/test/reference_evaluator_test.py"= ["C408"]  # dict(...) -> { ... }
```

Comment:
I think we want to keep this because those dictionaries are kwargs. Using a dict constructor preserves the kwarg syntax